packages/pn-personafisica-webapp/src/component/Notifications/FilterNotificationsFormBody.tsx

```diff
@@ -40,12 +40,34 @@ const FilterNotificationsFormBody = ({
   const { t } = useTranslation(['notifiche']);
   const isMobile = useIsMobile();
 
+  const handleDashInputMask = (input: string) => {
+    if(input.length){
+      // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+      void formikInstance.setFieldValue('iunMatch',formikInstance.values.iunMatch + input + '-');
+    }
+  };
+  
   const handleChangeTouched = (e: ChangeEvent) => {
     void formikInstance.setFieldTouched(e.target.id, true, false);
     formikInstance.handleChange(e);
```

Comment:
formik.handleChange e formik.setFieldValue aggiornano entrambi i valori della form. Evitare di chiamarli entrambi.

Reply:
in questo caso mi sembra servano entrambi, con l'handleChange tengo traccia dell'input dell'utente e con il setfieldvalue aggiungo la mia inputmask. Se togliessi uno dei due l'input mask non funzionerebbe più

Reply:
Questa cosa mi sembra strana. Lo scopo penso sia avere il valore dello iun senza i -, e avere a schermo lo iun con i -.
Non mi torna allora la logica, perchè sia con handleChange e con setFieldValue si va a modificare il valore del campo.
Di conseguenza mi aspetterei che nel momento in cui venga fatta una ricerca, si mandi al be lo iun con i -.
Provare con più ricerche consecutive